docs/reference/data.rst

```diff
@@ -171,15 +171,10 @@ When checking containment of an `Object` in a 2D region, Scenic will atuomatical
 
 3D regions represent points in 3D space.
 
-Most 3D regions inherit from a descendant of `MeshRegion` (an abstract class), either `MeshVolumeRegion` or `MeshSurfaceRegion`. These represent the volume (of a watertight mesh) or the surface of a mesh respectively. Various region classes are also provided to create primitive shapes. `MeshVolumeRegion` can be converted to `MeshSurfaceRegion` (and vice versa) using the the ``getSurfaceRegion`` and ``getVolumeRegion`` methods.
+Most 3D regions inherit from a descendant of `MeshRegion` (an abstract class that can't be directly instantiated), either `MeshVolumeRegion` or `MeshSurfaceRegion`. These represent the volume (of a watertight mesh) or the surface of a mesh respectively. Various region classes are also provided to create primitive shapes. `MeshVolumeRegion` can be converted to `MeshSurfaceRegion` (and vice versa) using the the ``getSurfaceRegion`` and ``getVolumeRegion`` methods.
```

Comment:
If we're not going to document `MeshRegion`, I don't think we should mention it here -- it might confuse people. The full documentation of `MeshVolumeRegion` and `MeshSurfaceRegion` will refer to it of course. (If there were lots of public APIs shared between the two subclasses that aren't available for general regions then that would be an argument for documenting them under `MeshRegion`, but I don't think that's the case.)